.travis.yml

```diff
@@ -24,28 +24,26 @@ matrix:
     - php: 5.4
       env: SYMFONY_VERSION=2.8.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.* CS_FIXER=run
+      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=strict CS_FIXER=run
```

Comment:
Please just remove this line instead: https://github.com/sonata-project/SonataMediaBundle/pull/913/files#diff-354f30a63fb0907d4ad57269548329e3R16